lib/resources/content/webpack.config.template.js

```diff
@@ -55,6 +54,58 @@ module.exports = ({production, server, extractCss, coverage, analyze, karma} = {
     sourceMapFilename: production ? '[name].[chunkhash].bundle.map' : '[name].[hash].bundle.map',
     chunkFilename: production ? '[name].[chunkhash].chunk.js' : '[name].[hash].chunk.js'
   },
+  optimization: {
+    // Use splitChunks to breakdown the vendor bundle into smaller files
+    // https://webpack.js.org/plugins/split-chunks-plugin/
+    splitChunks: { 
+      chunks: "initial",
+      cacheGroups: {
+        default: false, // Disable the built-in groups (default and vendors)
+        vendors: false,
+        bluebird: {
+          test: /[\\/]node_modules[\\/]bluebird[\\/]/,
+          name: "vendor.bluebird",
+          enforce: true,
+          priority: 100
+        },
+        // You can insert additional entries here for jQuery and bootstrap etc. if you need them
+        // Break the Aurelia bundle down into smaller chunks, binding and templating are the largest
+        aureliaBinding: {
```

Comment:
Could you elaborate why the separate of aurelia-binding and aurelia-templating into their own bundle? @chrisckc

Reply:
To provide a better chance of the app downloading successfully on an unreliable 3g connection the file sizes should be kept small.  Templating and binding are the biggest modules so splitting them out reduces the aurelia bundle size.

By splitting out vendor bundles you can take more advantage of the browser caching and webpack's chunk hashing to prevent re-downloading of unchanged modules when the app is updated and published to production.

The issue of making more http requests is outweighed by the advantages in my opinion, especially when using HTTP2.

While on this topic, it would be great if the webpack skeleton samples contained more examples of lazy loading.

Reply:
I see. This is nice & thanks for the explanation @chrisckc . 

Changes lgtm @EisenbergEffect . No one has complained about bulky single bundle so far, so I don't it's not well known optimization, or unnecessary or everyone has their own requirement of optimization. Regardless, I think this is good to promote as default behavior.